source/designers-guide/responsive-images/index.md

```diff
@@ -13,35 +13,51 @@ menu_order: 50
+
+This example is the Shopware equivalent to the basic `<img src="">` tag. Instead of applying the `src` attribute, we now have the opportunity to use the `srcset`, which allows us to specify the original thumbnail and its high dpi optimized version in the same attribute. Depending on the screen density, one of the two pictures is displayed. In convention the high dpi display image names are suffixed with a `@2x`.
 
 ```html
 <img srcset="product.jpg, product@2x.jpg 2x" alt="Produktfoto">
 ```
 
 ### Picture element
-The `picture` element takes the term "responsive" even a little step further. It is part of the [official W3C standard](http://www.w3.org/html/wg/drafts/html/master/semantics.html#the-picture-element "W3C picture element specifications") and is supported by most of the latest browsers. To ensure support for outdated browsers we are using the `picturefill.js` polyfill. Inside the `picture` element we can define multiple image sources and determine their visibility using a media query like syntax in the `media` attribute. In other words, we can choose which image is displayed at which viewport size. This fits perfectly well with the thumbnails that the thumbnail generator is able to create. The `picture` element also contains a default `img` tag, in cases the `media` attributes do not match the browser viewport.
+
+The `picture` element takes the term "responsive" even a little step further. It is part of the [official W3C standard](http://www.w3.org/html/wg/drafts/html/master/semantics.html#the-picture-element "W3C picture element specifications") and is supported by most of the latest browsers. To ensure support for outdated browsers we are using the `picturefill.js` polyfill. 
+
+Inside the `picture` element we can define multiple image sources and determine their visibility using a media query like syntax in the `media` attribute. In other words, we can choose which image is displayed at which viewport size or device pixel density. This fits perfectly well with the thumbnails that the thumbnail generator is able to create.
+
+Additionally, the `sizes` attribute defines how the image is shown in the different viewports. A common unit is `vw` which stands for *viewport with* and is the percentual width relative to the viewport.
+
+The `picture` element also contains a default `img` tag, in cases the `media` attributes do not match the browser viewport. The `src` of the fallback image should be the largest thumbnail available.
 
 ```html
 <picture>
-    <source media="(min-width: 64em)" srcset="product-large.jpg, product-large@2x.jpg 2px">
-    <source media="(min-width: 48em)" srcset="product-medium.jpg, product-medium@2px.jpg 2px">
-    <img srcset="product-small.jpg, product-small@2x.jpg 2x" alt="Produktfoto">
+    <source sizes="(min-width: 48em) 80vw, (min-width: 64em) 50vw, 100vw"
```

Comment:
Have you tested this? So is it realy the high dpi size you have to add and not the original one like 200w, 600w ... ?